crates/re_types_core/src/datatypes/float32_ext.rs

```diff
@@ -1,5 +1,11 @@
 use super::Float32;
 
+impl std::fmt::Display for Float32 {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:.prec$}", self.0, prec = crate::DISPLAY_PRECISION)
```

Comment:
Let not force a precision if the user has picked one already

```suggestion
        let precision = f.precision().unwrap_or(crate::DEFAULT_DISPLAY_DECIMALS);
        write!(f, "{:.prec$}", self.0, prec = precision)
```

Here and everywhere.


Also: `.$` picks the number of _decimals_, not the precision (`12.345` has 3 decimals, and a precision of 5 digits)

Reply:
as if I wrote any original code in this PR ;). That's c&p from Vec & friends.
Fixing it in more places then

Reply:
(very good change though! 👍 )